### ISSUE TITLE
Duplicate Key error

### ISSUE THREAD

**Issue by tpill90** [tpill90/steam-lancache-prefill#51]
```
Please enter your Steam password. (Password won't be saved) : *********
This account is protected by Steam Guard.  Please enter the code sent to your email address: *****
[12:54:50 PM] Logged into Steam
System.ArgumentException: An item with the same key has already been added. Key: 414170
```

```
System.ArgumentException: An item with the same key has already been added. Key: 414170
  at bool System.Collections.Generic.Dictionary`2.TryInsert(TKey , TValue , InsertionBehavior )
  at Dictionary<TKey, TElement> System.Linq.Enumerable.ToDictionary<TSource,TKey,TElement>(IEnumerable<TSource> ,
     Func<TSource, TKey> , Func<TSource, TElement> , IEqualityComparer<TKey> )
  at void SteamPrefill.Handlers.Steam.Steam3Session.LoadPackageInfo(IReadOnlyCollection<License> licenseList)
  at void SteamPrefill.Handlers.Steam.Steam3Session.LicenseListCallback(LicenseListCallback licenseList)
  at void SteamKit2.Internal.Callback`1.Run(object callback)
  at void SteamKit2.CallbackManager.<>c__DisplayClass10_0.<Handle>b__1(CallbackBase callback)
  at void System.Collections.Generic.List`1.ForEach(Action<T> )
  at void SteamKit2.CallbackManager.Handle(ICallbackMsg call)
  at void SteamKit2.CallbackManager.RunWaitAllCallbacks(TimeSpan timeout)
  at void SteamPrefill.Handlers.Steam.Steam3Session.TryWaitForLoginKey()
  at void SteamPrefill.Handlers.Steam.Steam3Session.<LoginToSteam>b__20_0(StatusContext _)
  at Task Spectre.Console.Status.<>c__DisplayClass14_0.<Start>b__0(StatusContext ctx) in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Status
     \Status.cs:44
  at void Spectre.Console.Status.<>c__DisplayClass16_0.<<StartAsync>b__0>d.MoveNext() in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Status
     \Status.cs:79
  at void Spectre.Console.Status.<>c__DisplayClass17_0`1.<<StartAsync>b__0>d.MoveNext() in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Status
     \Status.cs:120
  at void Spectre.Console.Progress.<>c__DisplayClass28_0`1.<<StartAsync>b__0>d.MoveNext() in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Progre
     ss\Progress.cs:133
  at async Task<T> Spectre.Console.Internal.DefaultExclusivityMode.RunAsync<T>(Func<Task<T>> func) in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Internal\
     DefaultExclusivityMode.cs:40
  at async Task<T> Spectre.Console.Progress.StartAsync<T>(Func<ProgressContext, Task<T>> action) in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Progre
     ss\Progress.cs:116
  at async Task<T> Spectre.Console.Status.StartAsync<T>(string status, Func<StatusContext, Task<T>> func) in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Status
     \Status.cs:117
  at async Task Spectre.Console.Status.StartAsync(string status, Func<StatusContext, Task> action) in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Status
     \Status.cs:77
  at void Spectre.Console.Status.Start(string status, Action<StatusContext> action) in
     C:\Users\Tim\Dropbox\Programming\dotnet-public\SteamPrefill\External\SpectreConsole\src\Spectre.Console\Live\Status
     \Status.cs:48
  at void SteamPrefill.Handlers.Steam.Steam3Session.LoginToSteam()
  at void SteamPrefill.SteamManager.Initialize()
  at async ValueTask SteamPrefill.CliCommands.SelectAppsCommand.ExecuteAsync(IConsole console)
  ```

**Comment by tpill90** [comment]
[SteamPrefill-1.2.1-osx-x64.zip](https://github.com/tpill90/steam-lancache-prefill/files/9251577/SteamPrefill-1.2.1-osx-x64.zip)
[SteamPrefill-1.2.1-win-x64.zip](https://github.com/tpill90/steam-lancache-prefill/files/9251579/SteamPrefill-1.2.1-win-x64.zip)
[SteamPrefill-1.2.1-linux-x64.zip](https://github.com/tpill90/steam-lancache-prefill/files/9251581/SteamPrefill-1.2.1-linux-x64.zip)

**Comment by tpill90** [comment]
Fix merged in #52

**Comment by tpill90** [comment]
Released in v1.2.2
